sorc/ncep_post.fd/WETBULB.f

```diff
@@ -59,13 +60,14 @@ SUBROUTINE WETBULB(T,Q,PMID,HTM,KARR,TWET)
 !!$omp&         presk,qbtk,qqk,sqk,sqs00k,sqs10k,tbtk,thesp,tpspk,
 !!$omp&         tqk,tthbtk,tthk)
 !-----------------------------------------------------------------------
-                             DO 300 L=1,LM
+      DO 300 L=1,LM
       DO 125 J=JSTA,JEND
       DO 125 I=1,IM
         IF (HTM(I,J,L)<1.0) THEN
           THESP(I,J)=273.15
           cycle    
         ENDIF
+        IF(T(I,J,L)<spval)THEN
```

Comment:
suggest add q and pmid

Reply:
@HuiyaChuang-NOAA Basically for these areas not in model computation domain, if T is undefined, Q, P, U/V will be the same values. But adding more constraints would be safe. Thanks!